src/stories/components/Forms/Select/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+
+import { SelectChangeEvent } from '@mui/material/Select';
+import { Option } from 'stories/components/Forms/Select/types/Option';
+import { Props } from 'stories/components/Forms/Select/types/Props';
+
+import * as S from './styles';
+
+export const Select: React.FC<Props> = ({ options, helperText, ...props }) => {
+  const [value, setValue] = useState<Option['value']>('');
+
+  function onChange(event: SelectChangeEvent<Option['value']>) {
+    setValue(event.target.value);
+  }
+
+  return (
+    <S.Wrapper>
+      {props.label && <S.Label>{props.label}</S.Label>}
+
+      <S.Select
+        value={value}
+        onChange={onChange}
+        label={props.label}
+        notched={!!props.label}
+        {...props}
+      >
+        <S.Option value="">Selecione uma opção</S.Option>
+
+        {options.map((option, index) => (
+          <S.Option key={`${option.value}-${index}`} value={option.value}>
+            {option.label}
+          </S.Option>
+        ))}
+      </S.Select>
+
+      {helperText && <S.HelperText>{helperText}</S.HelperText>}
```

Comment:
Sugiro passar um prop `error` ou alguma coisa que possa indicar que ocorreu um erro, para mudar a cor do texto `helperText` já que ele pode ser somente um texto informativo ou um texto de erro quando for um campo obrigatório por exemplo

Reply:
Por estar utilizando FormControl, automaticamente ao aplicar error ele mudará a cor do helper text

<img width="991" alt="Captura de Tela 2023-06-02 às 10 44 37" src="https://github.com/flow-build/studio-boilerplate/assets/72457264/68bae05f-68f4-4459-ab75-4880469ca788">